src/components/mod.rs

```diff
@@ -22,6 +22,17 @@
 // terms of the Steamworks API by Valve Corporation, the licensors of this
 // Program grant you additional permission to convey the resulting work.
 
+#[cfg(not(target_arch = "wasm32"))]
+/// Whether or not to use push constants when rendering the map editor. Disabling this will switch
+/// to fallback rendering using uniforms, which is slightly slower but is required for Luminol to
+/// work in web browsers until push constants are standardized in WebGPU.
+pub const USE_PUSH_CONSTANTS: bool = true;
+#[cfg(target_arch = "wasm32")]
+/// Whether or not to use push constants when rendering the map editor. Disabling this will switch
+/// to fallback rendering using uniforms, which is slightly slower but is required for Luminol to
+/// work in web browsers until push constants are standardized in WebGPU.
+pub const USE_PUSH_CONSTANTS: bool = false;
```

Comment:
Rather than a using a platform defined constant detecting if push constants are supported at runtime (using `wgpu::Adapter::features`) would allow luminol to work on platforms that don't support push constants